pontoon/pretranslation/tasks.py

```diff
@@ -2,18 +2,22 @@
 
 from django.db.models import Q, CharField, Value as V
 from django.db.models.functions import Concat
+from django.conf import settings
 
 from pontoon.base.models import Entity, TranslatedResource, Translation
 from pontoon.pretranslation.pretranslate import (
     get_translations,
     update_changed_instances,
 )
+from pontoon.base.tasks import PontoonTask
+from pontoon.sync.core import serial_task
 
 
 log = logging.getLogger(__name__)
 
 
-def pretranslate(project, locales=None, entities=None):
+@serial_task(settings.SYNC_TASK_TIMEOUT, base=PontoonTask, lock_key="project={0}")
```

Comment:
I'm getting the following warning when I start the pretranslate process while another pretranslate process for the same project is already running:

```
/usr/local/lib/python3.7/site-packages/django/core/cache/backends/base.py:245:
CacheKeyWarning: Cache key contains characters that will cause errors if used with memcached:
':1:serial_task.pontoon.pretranslation.tasks.pretranslate[project=Pontoon Test]'
  'used with memcached: %r' % key, CacheKeyWarning
```

That's because `{0}` gets replaced with the representation of the Project instance (i.e. project.name), which in my case includes space.

In the case of sync task, we pass the project PK.

We should either pass the project PK here as well or remove white spaces from the lock in the `serial_task()` function.

Reply:
Moving forward with passing project pk.